src/components/JUPTable/JUPTable.tsx

```diff
@@ -13,32 +14,23 @@ import {
   TableSortLabel,
   Typography,
 } from "@mui/material";
+import { TransitionGroup } from "react-transition-group";
 import SLink from "components/SLink";
-import { Data, IHeadCellProps } from "views/Dashboard/components/Widgets/TransactionsWidget/TransactionsWidget";
-import { ITransaction } from "types/NXTAPI";
 import { visuallyHidden } from "@mui/utils";
-import { DefaultTableRowsPerPage, TableRowsPerPageOptions } from "utils/common/constants";
-
-// might still want to use this concept
-// function createWidgetRow(date: string, qty: number, toFrom: string): Data {
-//   return {
-//     date,
-//     qty,
-//     toFrom,
-//   };
-// }
+import { DefaultTableRowsPerPage, DefaultTransitionTime, TableRowsPerPageOptions } from "utils/common/constants";
+import { IHeadCellProps, ITableRow } from ".";
 
 interface ITableTitleProps {
   title: string;
-  path: string;
+  path?: string;
   DisplayedComponents?: Array<React.ReactElement>;
 }
 
 const TableTitle: React.FC<ITableTitleProps> = ({ title, path, DisplayedComponents }) => {
   return (
     <>
       <TitleText variant="h6" id="tableTitle">
-        <SLink href={path}>{title}</SLink>
+        {path && <SLink href={path}>{title}</SLink>}
```

Comment:
I think my original logic did display just a title with no link (that was my goal). But linking to the full page as it does now doesn't hurt so i'll remove this.

Reply:
ah now I remember, I used it in the address book because there's not going to be an address book view/page and the dialog handles setting a title for what the dialog is for, so the table doesn't need to repeat that title information. 

i made title optional and refactored accordingly